perllib/FixMyStreet/App.pm

```diff
@@ -128,11 +129,9 @@ after 'prepare_headers' => sub {
 __PACKAGE__->log->disable('debug')    #
   unless __PACKAGE__->debug;
 
-# Check upload_dir
-my $cache_dir = path(FixMyStreet->config('UPLOAD_DIR'))->absolute(FixMyStreet->path_to());
-$cache_dir->mkpath;
-unless ( -d $cache_dir && -w $cache_dir ) {
-    warn "\x1b[31mCan't find/write to photo cache directory '$cache_dir'\x1b[0m\n";
+# Set up photo storage
+unless ( FixMyStreet::PhotoStorage::backend->init() ) {
```

Comment:
Wonder if it's a bit odd that this checks against S3 each start for the bucket's existence. I realise that matches the only-on-startup check the local upload dir exists, and you wouldn't want to do it all the time, but I'm not sure I'd expect just starting the app to do an external S3 call. Probably doesn't matter.

Reply:
I thought this was a better place to do the bucket check/creation rather than when a photo is uploaded as the latter is a more real time operation so the sooner it completes the better.